src/Sulu/Bundle/AdminBundle/Controller/TemplateController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This Controller returns javascript templates available from the AdminBundle.
```

Comment:
I think I don't like these comments on the controller anymore... If I check the other controllers they have no value at all and a few are already missing it. However, put it here for consistency.